Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:13.12.0-alpine
+
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package.json .
+COPY package-lock.json .
+
+RUN npm install --silent
+RUN npm install react-scripts@3.4.1 -g --silent
```

Comment:
Why are we installing a previous version of react-scripts globally in the dockerfile?

Reply:
Apologies, forgot to remove it when I copied and pasted the Dockerfile from a previous project

This has been completed.